packages/shared/src/utils/helpers/format-response.ts

```diff
@@ -47,13 +47,16 @@ export type TIDVErrorStatus = typeof idv_error_statuses[keyof typeof idv_error_s
 
 //formatIDVError is parsing errors messages from BE (strings) and returns error codes for using it on FE
 export const formatIDVError = (errors: string[], status_code: string) => {
-    if (errors.length === 0 && status_code === 'none') return null;
+    /**
+     * Check required incase of DIEL client
+     */
+    if (errors.length === 0 && [STATUS_CODES.NONE, STATUS_CODES.VERIFIED].includes(status_code)) return null;
     const error_keys: Record<string, TIDVErrorStatus> = {
         name: 'POI_NAME_MISMATCH',
         birth: 'POI_DOB_MISMATCH',
         rejected: 'POI_FAILED',
```

Comment:
maybe we can move this to constant file.
 You can ignore this comment as well 🙂

Reply:
@amina-deriv, thanks for suggestion. 
there is a card for BE improvements regarding idv errors. We will refactor this function later once BE is ready